lib/class-wp-rest-block-editor-global-styles-settings-controller.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * REST API: WP_REST_Block_Editor_Global_Styles_Settings_Controller class
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ */
+
+/**
+ * Core class used to retrieve the block editor global styles settings via the REST API.
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Block_Editor_Global_Styles_Settings_Controller extends WP_REST_Controller {
+	/**
+	 * Constructs the controller.
+	 */
+	public function __construct() {
+		$this->namespace = 'wp-block-editor/v1';
```

Comment:
```suggestion
		$this->namespace = 'wp/v2';
```

Reply:
Hey there 👋 the initial PR had this naming but it was [suggested](https://github.com/WordPress/gutenberg/pull/29969#issuecomment-802137951) to change it to follow a previously implemented endpoint for the block editor. What do you think? Thanks!